src/OpenSage.Game/GameContext.cs

```diff
@@ -51,7 +54,8 @@ internal GameContext(
             Navigation.Navigation navigation,
             Radar radar,
             IQuadtree<GameObject> quadtree,
-            Scene3D scene)
+            Scene3D scene,
```

Comment:
Van we remove Scene3D, or are there too many external usages still?

Reply:
Still too many usages. It's something I want to refactor though.